src/Application.cpp

```diff
@@ -24,6 +24,14 @@ static void stopHandler(int sig) {
     running = false;
 }
 
+/*
+    This method is created to avoid unnecessary calls of .._ALLOC methods from open62541 library just to satisfy parameter restrictions.
+    After this conversion regular methods like UA_NODEID_STRING can be used instead of .._ALLOC version.
+*/
+static char* convertToCStrNoConst(const std::string &tempStr) {
+    return const_cast<char*>(tempStr.c_str());
+}
+
```

Comment:
I don't think function is required for just const cast. Casting can be done in place.

Reply:
Yeah. Not a big difference in length. Switched to direct cast now.